app/src/renderer/components/staking/TableValidators.vue

```diff
@@ -39,100 +37,102 @@ export default {
     num: num,
     query: ``,
     sort: {
-      property: `percent_of_vote`,
-      order: `desc`
-    }
+      property: `commission`,
+      order: `asc`
+    },
+    rollingWindow: 10000 // param of slashing period
   }),
   computed: {
     ...mapGetters([
-      `delegation`,
       `committedDelegations`,
       `session`,
-      `liquidAtoms`,
-      `connected`,
+      `distribution`,
       `bondDenom`,
-      `keybase`
+      `keybase`,
+      `pool`
     ]),
-    vpTotal() {
-      return this.validators
-        .slice(0)
-        .map(v => {
-          v.voting_power = v.voting_power ? Number(v.voting_power) : 0
-          return v
-        })
-        .sort((a, b) => b.voting_power - a.voting_power)
-        .slice(0, 100)
-        .reduce((sum, v) => sum + v.voting_power, 0)
-    },
-    enrichedDelegates() {
-      return this.validators.map(v =>
+    enrichedValidators({
```

Comment:
this should be happening somewhere else / refactored (#1380 #2306) but moving it would be a big task so i just made sure it worked and was tested... because previously it didn't work and wasn't tested.